### PR TITLE
Completed first two exercises

### PR DESCRIPTION
**Pull request by nlefrancois6**

### CODE REVIEW

**Comment by igendr1** [comment]
wow great work

File: Lab02/CodingLab02_CosmologicalDistance.ipynb [hunk 1]
```diff
@@ -62,12 +62,10 @@
...
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "## Your answer here"
+    "ANSWER: This should be a reasonable assumption when the distance is large enough that H_0*d >> v_g , where H_0*d is the velocity due to expansion of the Universe and v_g is the velocity of the galaxy's movement due to all other effects. In other words, when the velocity due to expansion is the dominant term in the overall velocity."
```
Comment (igendr1) [inline Lab02/CodingLab02_CosmologicalDistance.ipynb:68]: Really good answer, i agree

Comment (katiesavard) [reply Lab02/CodingLab02_CosmologicalDistance.ipynb:68]: me too, good stuff, Noah

**Review by katiesavard — APPROVED** [review]
This is a good answer, good job!

**Pull request merged**